includes/after_build.yml

```diff
@@ -12,7 +12,7 @@ run:
           SiteSetting.disable_emails = "no"
         end
 
-        if MOZILLA_DISCOURSE_ENV == :staging
+        if MOZILLA_DISCOURSE_ENV == :stage
```

Comment:
Is there a corresponding PR/commits in the other repo updating this environment variable?

Reply:
yup, it's here https://github.com/mozilla-it/discourse-infra/pull/61
Terraform has been applied already so all the resources are in there.